f5-bigip-mixin/README.md

```diff
@@ -0,0 +1,175 @@
+# F5 BIG IP Mixin
+
+F5 BIG IP mixin is a set of configurable Grafana dashboards and alerts.
+
+The F5 mixin contains the following dashboards:
```

Comment:
Just thinking, when abbreviating how we refer to "F5 Big IP" is it more appropriate to say "Big IP" rather than "F5"? Should these dashboards be "Big IP cluster overview", "Big IP node overview", etc. 

For example, for Apache Tomcat we wouldn't say "Apache overview" if we shortened it. We'd say "Tomcat overview" right?

Reply:
Yeah, that makes sense. Updating F5 to Big IP or bigip when F5 is used alone.